event-management-backend/src/test/java/ro/msg/event_management/AdminHomepageIntegrationTests.java

```diff
@@ -56,6 +57,17 @@
     private static final LocalDate MAX_DATE = LocalDate.parse("2999-12-31");
     private static final LocalDate MIN_DATE = LocalDate.parse("1900-01-01");
 
+
+    @BeforeEach
+    void setUp() {
+        this.eventRepository.deleteAll();
+        this.sublocationRepository.deleteAll();
+        this.eventSublocationRepository.deleteAll();
+        this.locationRepository.deleteAll();
+        this.bookingRepository.deleteAll();
+        this.ticketRepository.deleteAll();
+    }
+
     @Test
     public void test_card_with_events() {
```

Comment:
From your side is a good temporary fix. The issue if with this test is that is not that descriptive. Right now it technically only test if there is an event and if that event is before or equal with himself.

Reply:
Changed the method name to a more descriptive one.